src/sentry/static/sentry/app/utils/performance/quickTrace/utils.ts

```diff
@@ -8,7 +8,7 @@ import {DiscoverQueryProps} from 'app/utils/discover/genericDiscoverQuery';
 import {
   EventLite,
   QuickTrace,
-  TraceFull,
+  QuickTraceEvent,
```

Comment:
I believe all the inputs in this file can be kept as `TraceFull` and you'll able to remove all the `?` and it should still work.